### PR TITLE
Fix issue #21

### PR DESCRIPTION
**Pull request by seetea**
Fix issue #21 to make the example can be built under both python 2.7 and python 3.4.

### CODE REVIEW

File: tensorflow_serving/session_bundle/example/export_half_plus_two.py [hunk 1]
```diff
@@ -63,7 +65,7 @@ def Export():
     # CopyAssets is used as a callback during export to copy files to the
     # given export directory.
     def CopyAssets(export_path):
-      print "copying asset files to: %s" % export_path
+      print ("copying asset files to: %s" % export_path)
```
Comment (kirilg) [inline tensorflow_serving/session_bundle/example/export_half_plus_two.py]: Please remove space before '('

**Comment by kirilg** [comment]
Looks good overall, just one minor comment. In addition to that, please sign the CLA (instructions given by googlebot) and squash all commits after you apply the suggested change. Thanks.

**Comment by seetea** [comment]
Kirilg, thank you for your suggestions very much. But I'm sorry that I'm still waiting for my company's permission to sign the CLA. Could you please wait for a few days? 

Anyway, I think I can make the commits meet your suggestions first.

I have squashed all the commits into the [Fix issue #21,support both python 2.7 and python 3.4](https://github.com/tensorflow/serving/pull/23/commits/12975054b9f1890a7c6cb8e96291bda7c803f2d5). Do you think it's OK now?

**Comment by kirilg** [comment]
No problem. Please take a look at https://github.com/tensorflow/serving/blob/master/CONTRIBUTING.md (namely the part that says "If you work for a company that wants to allow you to contribute your work, then you'll need to sign a corporate CLA." along with a link to the corporate CLA page, which I'm guessing is what you did already). Just ping back on this PR once the licensing is sorted out, and I'll merge.

With regards to the commits, I still see multiple commits in the "commits" tab (8). Please squash so there's only one.

**Comment by seetea** [comment]
Kirilg, I think I have made some mistakes when I squashed the commits. The PR was closed and recreated at https://github.com/tensorflow/serving/pull/26. Maybe we can continue the discussion there. Thanks.

**Pull request closed without merging**